docs/src/app/components/prop-type-description.css

```diff
@@ -41,3 +41,11 @@
   font-size: 95%;
   min-width: 250px;
 }
+
+.propTypeDescription  .requiredProp {
```

Comment:
That's a decendent selector. (Although I see an extra space in there from when I tried direct child that I'll clean up.) - Or did you mean scss / sass nesting?